src/gtk-3.0/gtk-widgets.css

```diff
@@ -249,7 +279,7 @@ combobox button:first-child arrow {
  *********/
 
 menubar {
-	border-image: url("assets/bump.png") 2 / 2px stretch;
+	border-image: url("assets/bump.svg") 2 / 2px stretch;
```

Comment:
How does this work? Is it 2px of the scaled image? 🤔 

I restarted my work on Flat-grey-rounded at https://github.com/puppylinux-woof-CE/gtk3_flat_grey_rounded/compare/master...puppylinux-woof-CE:gtk3_flat_grey_rounded:hidpi-v2?expand=1, and many things are broken. Maybe I should just convert all assets to SVG, like you did, then start converting some assets to CSS.

Reply:
Saw v2

I think there is no harm keeping svg assets. I've no idea how the `2 / 2px stretch` works !